.github/ISSUE_TEMPLATE/aws-access-request.yml

```diff
@@ -41,59 +41,65 @@ body:
         - Magnum Opus - Paula G. Mendoza
         - Liberty IT -  Michael G. Lovejoy
         - Accenture Federal Services - Alexa Elliot
-        - Other
+        - Other - please specify in 'Additional Notes'
```

Comment:
I like this 👍

Reply:
Like it: fixed in 208fa6b4843f130b49fff88d32d6165c58e67445